### PR TITLE
latency-tests: improve logs clarity

### PR DESCRIPTION
**Pull request by Tal-or**
A couple of log improvements for latency-tests to make it easier to debug in case of test failures.

### CODE REVIEW

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Comment by fromanirh** [comment]
please rebase on top of current master branch

File: test/e2e/performanceprofile/functests/4_latency/latency.go [hunk 1]
```diff
@@ -479,11 +480,10 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 	podTimeout := time.Duration(timeout + 120)
 	err = pods.WaitForPhase(testPod, corev1.PodSucceeded, podTimeout*time.Second)
 	if err != nil {
-		testlog.Error(err)
 		logEventsForPod(testPod)
 		testlog.Info(getLogFile(node, logName))
 	}
-	Expect(err).ToNot(HaveOccurred())
+	Expect(err).ToNot(HaveOccurred(), "pod %q did not reach %q phase; %v", podKey, corev1.PodSucceeded)
```
Comment (shajmakh) [inline test/e2e/performanceprofile/functests/4_latency/latency.go:486]: should also pass "err" to fill %v

Comment (Tal-or) [reply test/e2e/performanceprofile/functests/4_latency/latency.go]: Done. Thanks!

**Comment by shajmakh** [comment]
thanks!
/lgtm

**Pull request merged**